person.rb

```diff
@@ -1,5 +1,7 @@
-class Person
+require_relative './decorators/nameable'
```

Comment:
Your `person` class should require the capitalize_decorator and trimmer_decorator at the top to ensure the below error is fixed
![Screenshot from 2023-05-17 15-03-59](https://github.com/Caren-Koroeny/school-library/assets/44624138/baa53126-9375-4548-be51-d6d7487948e8)
Adding this should fix it
```
require_relative './decorators/capitalize_decorator'
require_relative './decorators/trimmer_decorator'
```